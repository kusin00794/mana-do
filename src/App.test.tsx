import { testSnapshots } from './utils/test';
import App from './App';

jest.mock('./pages/routes', () => 'Routes');

describe('<App />', () => {
  testSnapshots(App, [
    {
      description: 'render App',
      props: {},
    },
  ]);
});
