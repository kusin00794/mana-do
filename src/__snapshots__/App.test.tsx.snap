// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> snapshots render App 1`] = `<Routes />`;
